ui/src/utils/History.js

```diff
@@ -62,10 +62,15 @@ export const setHistory = (historyParams = {}) => {
 /**
  *
  * @param {string} query params name from url
+ * @param {any} default return default value in case the query not exists
  * @returns {string} Param value from url
  */
-export const getHistory = (query) => {
+export const getHistory = (query, defaultReturn = null) => {
   const searchParams = new window.URLSearchParams(window.location.search);
   const searchQuery = searchParams.get(query);
-  return searchQuery;
+  if (searchQuery) {
```

Comment:
return searchQuery || defaultValue;
is more readable

Reply:
I do not agree. this is more readable to me